apps/trading-e2e/src/integration/navbar.cy.ts

```diff
@@ -1,11 +1,13 @@
 import { mockConnectWallet } from '@vegaprotocol/cypress';
 
 beforeEach(() => {
+  cy.clearLocalStorage();
```

Comment:
Cypress should automatically clear local storage after each test so I don't think `cy.clearLocalStorage()` is needed.

Reply:
Good point